### PR TITLE
Format object columns according to API specs

### PR DESCRIPTION
**Pull request by amitlicht**
* Review different object columns and format them according to specs
* Support yaml output

### CODE REVIEW

File: src/pkg/output/formatters.go [hunk 1]
```diff
@@ -4,11 +4,23 @@ import (
 	"fmt"
 	"github.com/otterize/otterize-cli/src/pkg/cloudclient/restapi/cloudapi"
 	"github.com/samber/lo"
+	"golang.org/x/exp/constraints"
 	"strings"
 )
 
+func formatD[T constraints.Integer](c T) string {
```
Comment (orishoshan) [inline src/pkg/output/formatters.go:11]: wut why? this just makes it harder to see that all it does is stringify the integer

**Review by orishoshan — APPROVED** [review]

File: src/pkg/output/formatters.go (hunk 1 above)
Comment (amitlicht) [reply src/pkg/output/formatters.go:11]: because there's so many uses that duplicate it and the code is horrible :(

Comment (orishoshan) [reply src/pkg/output/formatters.go:11]: but it's fmt.Sprintf!
everyone knows what Sprintf does, and it's a single function call - which was replaced with another single function call, but then nobody knows what it does

Comment (amitlicht) [reply src/pkg/output/formatters.go:11]: It was replaced with a shorter function call... I felt like it improves readability but I'm not feeling too strong about it, so reverting it if you insist.

**Pull request merged**